optuna/integration/tfkeras.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import
+
+import optuna
+from optuna import type_checking
+
+if type_checking.TYPE_CHECKING:
+    from typing import Any  # NOQA
+    from typing import Dict  # NOQA
+
+try:
+    from tensorflow.keras.callbacks import Callback
+    _available = True
+except ImportError as e:
+    _import_error = e
+    # TFKerasPruningCallback is disabled because TensorFlow is not available.
+    _available = False
+    Callback = object
+
+
+class TFKerasPruningCallback(Callback):
+    """tf.keras callback to prune unpromising trials.
+
+    This callback is intend to be compatible for TensorFlow v1 and v2,
+    but only tested with TensorFlow v1.
+
+    Example:
+
+        Add a pruning callback which observes validation losses.
+
+        .. code::
+
+            model.fit(x, y, callbacks=[TFKerasPruningCallback(trial, 'val_loss')])
+
+    Args:
+        trial:
+            A :class: `~optuna.trial.Trial` corresponding to the current evaluation of the
```

Comment:
Please remove the whitespace between `:class:` and `~optuna.trial.Trial` to enable the link to `Trial` properly.

Reply:
I fixed it